amazon/ion/ion_py_objects.py

```diff
@@ -0,0 +1,586 @@
+from decimal import Decimal
+from collections.abc import MutableMapping
+
+from amazon.ion.core import IonType, IonEvent
+from amazon.ion.symbols import SymbolToken
+
+
+class IonPyNull_new(object):
+    __name__ = 'IonPyNull_new'
+    __qualname__ = 'IonPyNull_new'
+    ion_type = IonType.NULL
+
+    def __init__(self, ion_type=IonType.NULL, value=None, annotations=()):
+        self.ion_type = ion_type  # TODO initialized to NULL type first, what's the real type?
```

Comment:
what is the action here? I would say it's OK as it is... or suggest that for `IonPyNull` we could avoid the class attribute. But I don't understand if this points to a deeper issue.

Reply:
Removed.